R/canned-epipred.R

```diff
@@ -30,6 +31,21 @@ arx_lags_validator <- function(predictors, lags) {
       "You have requested {p} predictor(s) but {l} different lags.",
       i = "Lags must be a vector or a list with length == number of predictors."
     ))
+  }else{
+    if(length(lags) == sum(names(lags) != "", na.rm = TRUE)){
+      if(!all(names(lags) %in% predictors)){
```

Comment:
Similar linting comment.

Also, let's reverse this `if` `else` pair. First do the one that works else throw the error.